src/hotspot/share/code/stubs.cpp

```diff
@@ -217,8 +217,6 @@ void StubQueue::verify() {
   guarantee(0 <= _queue_begin  && _queue_begin  <  _buffer_limit, "_queue_begin out of bounds");
   guarantee(0 <= _queue_end    && _queue_end    <= _buffer_limit, "_queue_end   out of bounds");
   // verify alignment
-  guarantee(_buffer_size  % stub_alignment() == 0, "_buffer_size  not aligned");
-  guarantee(_buffer_limit % stub_alignment() == 0, "_buffer_limit not aligned");
```

Comment:
Why these were removed?

Reply:
I suggested to remove it, because it shouldn't be necessary to align the end of the code.  Only the data header and code start should need alignment.